src/pelib/ImageLoader.cpp

```diff
@@ -486,6 +486,50 @@ uint32_t PeLib::ImageLoader::getFileOffsetFromRva(uint32_t rva) const
 	return rva;
 }
 
+// similar to getFileOffsetFromRva, but the offset is within the real file and not memory image
+uint32_t PeLib::ImageLoader::getValidOffsetFromRva(uint32_t rva) const
```

Comment:
Please use `std::uint32_t` instead of just `uint32_t` to follow used standards in retdec.

Reply:
I haven't noticed this is pelib code which might use different code style, in that case disregard.

Reply:
Yeah, ImageLoader has them typedef'd at the top of source file with comment `// Anti-headache` so I just followed the used style within the unit

Reply:
Also regarding coding styles, just a thought - wouldn't it be better to unify the coding styles within the RetDec with some `.clang-format` rules?

Reply:
Please remove those `Anti-headache` defs, lets use `std::whatever` everywhere.

Reply:
Done